docs/source/getting_started/cmakepp_examples/index.rst

```diff
@@ -14,5 +14,6 @@ CMakePP automatically. If you have not done this step, please see
    :numbered:
 
    classes
+   exceptions
```

Comment:
Is the warning from the `docs_test` action valid? (does `getting_started/cmakepp_examples/exceptions` exist?)